rosidl_typesupport_opensplice_cpp/rosidl_typesupport_opensplice_cpp/__init__.py

```diff
@@ -30,14 +30,15 @@ def generate_dds_opensplice_cpp(
 
         cmd = [idl_pp]
         for include_dir in include_dirs:
-            cmd += ['-I', include_dir]
+            cmd += ['-I', '"%s"' % include_dir]
```

Comment:
I did not try those. I kept the resultant syntax the same with the space betwen -I and the value.